src/pages/FolderPage.jsx

```diff
@@ -45,9 +53,28 @@ function FolderPage() {
     const response = await getUser();
     const userInfo = response.data[0];
     setUserInfo(userInfo);
-    console.log(userInfo);
   };
 
+  const changeLinkDeleteSelect = useCallback(() => {
+    setIsLinkDeleteSelect(!isLinkDeleteSelect);
+  }, [isLinkDeleteSelect]);
+
+  const changeFolderDeleteSelect = useCallback(() => {
+    setIsFolderDeleteSelect(!isFolderDeleteSelect);
```

Comment:
그나저나 이것도 이렇게 바뀌어야 할 것 같아요.
```suggestion
    setIsFolderDeleteSelect((prev) => !prev);
```